src/main/java/uk/gov/hmcts/reform/et/syaapi/service/NotificationService.java

```diff
@@ -97,4 +109,76 @@ public SendEmailResponse sendSubmitCaseConfirmationEmail(CaseDetails caseDetails
         }
         return sendEmailResponse;
     }
+
+    /**
+     * Format details of claimant request and retrieve case data, then send email.
+     *
+     * @param caseDetails existing case details
+     * @param claimantApplication application request data
+     * @return Gov notify email format
+     */
+    public SendEmailResponse sendAcknowledgementEmailToClaimant(CaseDetails caseDetails,
+                                                                ClaimantTse claimantApplication) {
+
+        CaseData caseData = EmployeeObjectMapper.mapRequestCaseDataToCaseData(caseDetails.getData());
+        String claimant = caseData.getClaimantIndType().getClaimantFirstNames() + " "
+            + caseData.getClaimantIndType().getClaimantLastName();
+        String caseNumber = caseData.getEthosCaseReference();
+        String respondentNames = getRespondentNames(caseData);
+        String caseId = caseDetails.getId() == null ? "case id not found" : caseDetails.getId().toString();
+
+        SendEmailResponse sendEmailResponse;
+        Map<String, String> parameters = new ConcurrentHashMap<>();
+        parameters.put("claimant", claimant);
+        parameters.put("respondentNames", respondentNames);
+        parameters.put("caseId", caseId);
+        parameters.put("caseNumber", caseNumber);
+
+        String emailTemplate = notificationsProperties.getClaimantTseEmailTypeCTemplateId();
+        if (!TYPE_C.equals(claimantApplication.getContactApplicationType())) {
```

Comment:
Very complex method, could we simplify it a little

Reply:
Refactored and simplified